mitodl/open-discussions
# a few dependency upgrades, switch over a few reducers to use redux toolkit

#### Pre-Flight checklist




#### What are the relevant tickets?

no ticket, just a little tech debt stuff.

#### What's this PR do?

basically just a) updating webpack and related tooling and b) installing the reduxjs toolkit and using that to rewrite a few reducers.

#### How should this be manually tested?

I think click around the app and make sure nothing breaks - so test it thoroughly, go and make posts and comments, create a channel, try out the search, make sure podcasts is all working and you can play audio, etc etc.
